dracut/modules.d/90qubes-pciback/qubes-pciback.sh

```diff
@@ -56,7 +56,17 @@ for dev in $HIDE_PCI; do
     echo -n "$BDF" > /sys/bus/pci/drivers/pciback/new_slot
     echo -n "$BDF" > /sys/bus/pci/drivers/pciback/bind
 done
-) || die 'Cannot unbind PCI devices.'
+) || {
+    hypervisor_type=
+    if [ -r /sys/hypervisor/type ]; then
+        read -r hypervisor_type < /sys/hypervisor/type
```

Comment:
```suggestion
        read -r hypervisor_type < /sys/hypervisor/type || die 'Cannot determine hypervisor type'
```

Reply:
When you write such suggestion, you do realize this whole PR is about undoing your overly strict error checking?
(I can't think of a way why reading from existing /sys/hypervisor/type would fail, but also, can't miss the irony)

Reply:
:laughing: Yes, the irony is real